### PR TITLE
Adding assets from v6

### PR DESCRIPTION
**Pull request by dmitryzuev**
Add scss and js from v6
Version update

### CODE REVIEW

**Comment by decabeza** [comment]
:tada:

**Comment by danmitchell-** [comment]
+1 im waiting for this!

**Comment by victorsosa** [comment]
+1

**Comment by whatcould** [comment]
Just a note to those of you who want to use Foundation 6 with foundation-rails **right now** — I spent some time figuring this out, and hopefully can save you the work.

* This branch/PR from dmitryzuev does not reference the correct _settings.scss file (which becomes foundation_and_overrides), so you'd have to manually copy that file.
* Neither did the other most-recently-updated 6.0.x fork, CatchRelease/foundation-rails

My fork (https://github.com/whatcould/foundation-rails) *does* work if you want to use Foundation 6 now, and get the correct settings file. Just switch to that fork in your Gemfile, bundle, and run `rails g foundation:install`.

There were some changes in how the JS and SCSS were located within directories between 5 and 6, and I'm not sure how to reconcile those — I'm assuming the Zurb folks might have more of an idea of how they want to handle that. And these changes are so minor, I'm not sure it's worth a pull request on my part.

But my fork least gets you started on 6, and it seems like any changes they make will only require minor tweaks to the end user of the gem.

**Comment by dmitryzuev** [comment]
Thanks, @whatcould 
I had updated my branch

**Comment by mattvleming** [comment]
Also waiting for this

**Comment by csm123** [comment]
+1

File: vendor/assets/javascripts/foundation.js [hunk 1]
```diff
@@ -1,17 +1,28 @@
 //= require foundation/foundation.abide
 //= require foundation/foundation.accordion
-//= require foundation/foundation.alert
-//= require foundation/foundation.clearing
+//= require foundation/foundation.accordionMenu
+//= require foundation/foundation.core
```
Comment (DmitryBochkarev) [inline vendor/assets/javascripts/foundation.js]: order is not correct `foundation/foundation.core` should go first

**Comment by DmitryBochkarev** [comment]
https://github.com/dmitryzuev/foundation-rails/blob/feature-foundation6/vendor/assets/stylesheets/foundation.scss#L52 foundation 6 includes fluid grid, maybe `foundation-everything` shouldn't include fixed grid?

**Comment by whatcould** [comment]
Re: JavaScript order, use my fork! I carefully used the order from the Foundation grunt file.

**Comment by whatcould** [comment]
Re: fixed grid, the fluid grid use is going to be limited to people who don't have to support IE9 — which I guess they're assuming is a minority. the "foundation-everything" is just a crutch to get started quickly anyhow (and comes straight from the master repo), so it seems like the standard grid is the better choice there. I bet the logic is, anybody advanced enough to be using the flexgrid will probably be picking and choosing their components anyway.

**Comment by zhouguangming** [comment]
+1

**Comment by cavpollo** [comment]
+1

**Comment by weiserma** [comment]
@whatcould - I changed to your branch; bundle update; then rails g foundation:install

I get the following error:
File to import not found or unreadable: foundation/util/util

application.sccs:
```
/*
 * This is a manifest file that'll be compiled into application.css, which will include all the files
 * listed below.
 *
 * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
 * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
 *
 * You're free to add application-wide styles to this file and they'll appear at the bottom of the
 * compiled file so the styles you add here take precedence over styles defined in any styles
 * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
 * file per style scope.
 *
 *= require_tree .
 *= require chosen
 *= require_self
 //= require foundation_and_overrides

 *= require jquery-ui
 */

@import "font-awesome-sprockets";
@import "font-awesome";

@import 'foundation_and_overrides';  #tried foundation... same error
@include foundation-everything;
```

**Comment by drdevnull** [comment]
+1

**Comment by MitchellGeere** [comment]
I feel like I am missing something here. I am using https://github.com/whatcould/foundation-rails you know because "My fork (https://github.com/whatcould/foundation-rails) does work if you want to use Foundation 6 now". However although everything runs and util/util is not an issue etc. it simply loads nothing. http://cl.ly/eCUq 

I feel like I am missing something super obvious here.

**Comment by JCBarry** [comment]
+1

**Comment by pav31** [comment]
+1

**Comment by sartipaz** [comment]
+1

**Comment by arthurzurb** [comment]
Hey Everyone!

We are working on finishing up what we have on the F6-dev branch. We have been able to create a rake task to update the source code and that works great, currently we are working through what seems to be a Sass issue. sass-rails 5.0 is using Sass 3.4.20 which has deprecated the way variables are written. 

We are trying to get a workaround for this, or we might have to update the Foundation for Sites code and release it. If anyone has time to take a look and maybe has some suggestions just let me know and I will pass it along to the rest of the team. Thanks to everyone for your patience and contributions, we are a pretty small team and your help makes a ton of difference! We have been using a few pieces from these PRs.

Again, thanks everyone for supporting this gem. We have plans to keep it growing and we are open to any suggestions.

**Comment by arthurzurb** [comment]
UPDATE:
The new gem has been published!
Head over to ruby gems: https://rubygems.org/gems/foundation-rails/versions/6.1.1.0
Foundation for Sites 5 lives on the v5 branch, and Foundation for Sites 6 now lives on Master. Please feel free to install, test it out for your projects and of course, submit any issues & pull requests on this repo. Just a note, Sass is deprecating some things so you will see some of those warnings, however we are in the process of updating the Sass syntax to get rid of those & a patch will be out within the next couple of weeks. Thanks!

**Pull request closed without merging**